tests/integration/TEC/Tickets/Commerce/Admin/Singular_Order_PageTest.php

```diff
@@ -75,7 +75,27 @@ public function it_should_match_order_actions() {
 			ob_start();
 			$singular_page->render_actions( $order );
 			$html[] = str_replace( $order->ID, '{{order_id}}', ob_get_clean() );
+		}
+
+		$this->assertMatchesHtmlSnapshot( implode( PHP_EOL . '<!--NEXT ITEM-->' . PHP_EOL, $html ) );
```

Comment:
crap :facepalm: thank you!

Reply:
thats the reason i though the change was covered :(

Reply:
Yeah, no problem.